NSS-Day-Cohort-49/react-nutshell-props-pushers
# Friends group code

#### Changes Made

1. Added file `searchFriends.js` to `friends` directory.

   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout friends-group-code
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user does enters a search, then it should re render the new list in the browser should happen.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
